src/hierarchical_deterministic/bip39/bip39_word/bip39_word.rs

```diff
@@ -33,27 +33,28 @@ impl PartialOrd for BIP39Word {
 }
 
 impl BIP39Word {
-    pub fn new(word: &'static str, language: BIP39Language) -> Result<Self> {
+    pub fn new(word: impl AsRef<str>, language: BIP39Language) -> Result<Self> {
```

Comment:
It is nice to use `impl AsRef<str>` or `impl Into<String>` instead of `String` or `&str` since both can be passed!

This allows us to skip calling `.to_owned()` on a `&str`, but it also allows us to pass a `String` if we have one.